gr-fec/python/fec/qa_ecc_ccsds_27.py

```diff
@@ -21,9 +21,9 @@ def setUp(self):
     def tearDown(self):
         self.tb = None
 
-    def xtest_ccsds_27(self):
-        src_data = (1, 2, 3, 4, 5, 6, 7, 8, 9, 10)
-        expected = (0, 0, 0, 0, 1, 2, 3, 4, 5, 6)
+    def test_ccsds_27(self):
+        src_data = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+        expected = [0, 0, 0, 0, 1, 2, 3, 4, 5, 6]
```

Comment:
This test used to be disabled. The problem was a type mismatch (list vs. tuple), which I've solved by switching the inputs to lists to match the output type of the Vector Sink.